Sparkle/SPUCoreBasedUpdateDriver.m

```diff
@@ -285,6 +306,12 @@ - (void)installerDidStartInstalling
     }
 }
 
+- (void)installerDidStartExtracting
+{
+    // The installer has moved the archive and no longer needs the download directory
```

Comment:
Is it guaranteed to move at the start? Shouldn't it be deleted after the end in case the installer implementation still uses the file?

Reply:
The installer process moves the update from the download directory to its own cache directory before starting to extract it. So the framework no longer "owns" the download and is transferred. (This also prevents the framework from still accessing the download; think eg sandbox -> non-sandbox installer, user -> root installer) The download directory from the framework is safe to remove at this point.